fe1-web/src/features/evoting/objects/index.ts

```diff
@@ -1 +1,3 @@
 export * from './Election';
+export * from './Configuration';
+export * from './Feature';
```

Comment:
Not too comfortable about these here. I don't have a better idea for now, though.

Reply:
I agree that a different place would be nicer. What about features/evoting/ directly since this defines the evoting feature interface? Or a subfolder called interface?